src/skylink/sia.ts

```diff
@@ -50,15 +50,55 @@ export class SiaSkylink {
   toString(): string {
     return encodeSkylinkBase64(this.toBytes());
   }
+
+  /**
+   * Loads the given raw data and returns the result. Based on sl.LoadBytes in
+   * skyd.
+   *
+   * @param data - The raw bytes to load.
+   * @returns - The sia skylink.
+   * @throws - Will throw if the data has an unexpected size.
+   */
+  static fromBytes(data: Uint8Array): SiaSkylink {
+    // Sanity check the size of the given data.
+    if (data.length !== RAW_SKYLINK_SIZE) {
+      throw new Error("Failed to load skylink data");
+    }
+
+    const view = new DataView(data.buffer);
+
+    // Load the bitfield.
+    const bitfield = view.getUint16(0, true);
+    // TODO: Validate v1 bitfields.
```

Comment:
What is there to validate? Don't we expect the bitfield to be only zeros?

Reply:
`LoadBytes` in `skyd` here calls `validateAndParseV1Bitfield`, which looked annoying to implement. 😛 It probably isn't so bad, but it'd have to be tested, as I don't want there to be a bug that rejects valid bitfields.

Reply:
Oh, I see. I saw the function and it does check many things. I guess it's fine to add those later.